components/payments/settings/payments-webhooks-table.tsx

```diff
@@ -78,6 +90,9 @@ export const PaymentsWebhooksTable: React.FC<PaymentsWebhooksTableProps> = ({
   const editWebhookRef = React.useRef<HTMLInputElement>(null);
   const createWebhookRef = React.useRef<HTMLInputElement>(null);
 
+  const [webhookToRevoke, setWebhookToRevoke] = React.useState<TableWebhookType>();
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
   React.useEffect(() => {
```

Comment:
same for this, we shouldn't need to keep any state of what we're editing do we? can't we just pass it in to the component when opening the modal?